src/ui/components/-utils/todo.ts

```diff
@@ -0,0 +1,26 @@
+import { tracked } from "@glimmer/component";
+
+export default class Todo {
+  id: number;
```

Comment:
The `id` should be private. It's just a temporary hack because we don't have a good alternative to `key="id"` at the moment.